docs/csharp/programming-guide/arrays/index.md

```diff
@@ -2,51 +2,47 @@
 title: "Arrays - C# Programming Guide"
 ms.custom: seodec18
 ms.date: 07/20/2015
-helpviewer_keywords: 
+helpviewer_keywords:
   - "arrays [C#]"
   - "C# language, arrays"
 ms.assetid: bb79bdde-e570-4c30-adb0-1dd5759ae041
 ---
 # Arrays (C# Programming Guide)
 
-You can store multiple variables of the same type in an array data structure. You declare an array by specifying the type of its elements.  
-  
- `type[] arrayName;`  
-  
- The following example creates single-dimensional, multidimensional, and jagged arrays:  
-  
- [!code-csharp[csProgGuideArrays#1](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideArrays/CS/Arrays.cs#1)]  
-  
-## Array Overview
-
- An array has the following properties:  
-  
-- An array can be [Single-Dimensional](./single-dimensional-arrays.md), [Multidimensional](./multidimensional-arrays.md) or [Jagged](./jagged-arrays.md).  
-  
-- The number of dimensions and the length of each dimension are established when the array instance is created. These values can't be changed during the lifetime of the instance.  
-  
-- The default values of numeric array elements are set to zero, and reference elements are set to null.  
-  
-- A jagged array is an array of arrays, and therefore its elements are reference types and are initialized to `null`.  
-  
-- Arrays are zero indexed: an array with `n` elements is indexed from `0` to `n-1`.  
-  
-- Array elements can be of any type, including an array type.  
-  
-- Array types are [reference types](../../language-reference/keywords/reference-types.md) derived from the abstract base type <xref:System.Array>. Since this type implements <xref:System.Collections.IEnumerable> and <xref:System.Collections.Generic.IEnumerable%601>, you can use [foreach](../../language-reference/keywords/foreach-in.md) iteration on all arrays in C#.  
-  
-## Related Sections  
-  
-- [Arrays as Objects](./arrays-as-objects.md)  
-  
-- [Using foreach with Arrays](./using-foreach-with-arrays.md)  
-  
-- [Passing Arrays as Arguments](./passing-arrays-as-arguments.md)  
-  
-## C# Language Specification
-
- [!INCLUDE[CSharplangspec](~/includes/csharplangspec-md.md)]  
-  
+You can store multiple variables of the same type in an array data structure. You declare an array by specifying the type of its elements. If you want the array to store elements of different types, you can specify `object` as its type. In the unified type system of C#, all types, predefined and user-defined, reference types and value types, inherit directly or indirectly from <xref:System.Object>.
```

Comment:
Nit: I changed "different types" to "any type" because it would generally be best practice to name the most specific type possible for a use case. (for example a common base class for a specific use.) But, this article isn't a good place to go into a lot of detail on those concerns.

```suggestion
You can store multiple variables of the same type in an array data structure. You declare an array by specifying the type of its elements. If you want the array to store elements of any type, you can specify `object` as its type. In the unified type system of C#, all types, predefined and user-defined, reference types and value types, inherit directly or indirectly from <xref:System.Object>.
```